### ISSUE TITLE
when i register a type extension for a exist custom element,  must throw error but no.

### ISSUE THREAD

**Issue by acelan86** [WebReflection/document-register-element#47]
see http://w3c.github.io/webcomponents/spec/custom/#dfn-definition-construction-algorithm

> If NAME was provided and is not null:
 + Let BASE be the element interface for NAME and NAMESPACE
 + If BASE does not exist or is an interface for a custom element, set ERROR to InvalidName and stop.

when i register a type extension for a exist custom element,  must throw error (chrome is right because use intern registerElement, but other is wrong because that have no intern registerElement, use document-register-element.js)

```javascript
var XA = document.registerElement( 'x-a', {
      ....
});

// must throw error
var XD = document.registerElement( 'x-d', {
       'extends': 'x-a' 
});
```

**Comment by WebReflection** [comment]
it does:
https://github.com/WebReflection/document-register-element/blob/master/build/document-register-element.max.js#L635-L640

and it's tested too ...

**Comment by WebReflection** [comment]
Actually, an extends is not  a register. You are registering `x-d` that extends `x-a`, it's like registering `x-i` that extends `input` which is already know. Why should that fail?

**Comment by WebReflection** [comment]
fixed in `0.5.1` ... for what is worth my opinion, this seems like an unnecessary specs limitation.

**Comment by acelan86** [comment]
haha.. i think so.
but for a compatible scheme， it's must has same result in all browsers.

Thank u :) , for fixed
